scripts/archive.js

```diff
@@ -0,0 +1,61 @@
+const fs = require('fs-extra');
```

Comment:
Can we have a `#/usr/bin/env node` on every script ?

Reply:
It makes sense to do that on entry points script. For instance, `archive.js` is just a bunch of exported utility functions, so executing it is useless.

So I'll do that on every `job-*.js` files. (I will also `chmod +x` them)